jbpm-flow/src/main/java/org/jbpm/process/core/timer/Timer.java

```diff
@@ -32,7 +32,16 @@
     private String period;
     private String date;
     private int timeType;
+    private String name;
```

Comment:
Add name to the `toString` of this class

Reply:
done